tests/test_helpers.py

```diff
@@ -66,3 +68,59 @@ def test_local_model_non_default_name(self, tmp_path):
         # with adapter name
         result = check_if_peft_model(tmp_path / "peft-gpt2-other" / "other")
         assert result is True
+
```

Comment:
Please put this into a new test class with an appropriate name, as this is unrelated to `TestCheckIsPeftModel`. You can also define `scale_from_modules` inside of that class.

Reply:
Took care of this!